test-utils/runtime/Cargo.toml

```diff
@@ -63,16 +65,22 @@ sc-executor = { version = "0.10.0-dev", path = "../../client/executor" }
 sp-consensus = { version = "0.10.0-dev", path = "../../primitives/consensus/common" }
 substrate-test-runtime-client = { version = "2.0.0", path = "./client" }
 sp-tracing = { version = "10.0.0", path = "../../primitives/tracing" }
+json-patch = { version = "1.0.0", default-features = false }
 
 [build-dependencies]
 substrate-wasm-builder = { version = "5.0.0-dev", path = "../../utils/wasm-builder", optional = true }
 
 [features]
 default = [
 	"std",
+	"genesis-builder",
+]
+genesis-builder = [
```

Comment:
Why do we need this feature?

Reply:
It is a substrate-test-runtime so I added it to easily build a runtime with builder disabled. We could live w/o it, but I found it useful just for dev purposes: size comparison, error handling when builder is not present, etc...

Reply:
I was just asking as you currently enable the feature all the time in `build.rs`.

Reply:
Changed to `disable-genesis-config` + env variable in `build.rs` to disable GenesisConfig support.